nautobot/core/models/generics.py

```diff
@@ -82,6 +82,8 @@ class PrimaryModel(BaseModel, ChangeLoggedModel, CustomFieldModel, RelationshipM
     tangible or logical resources on the network, or within the organization.
     """
 
+    # TODO: why did we change this?
+    # tags = TaggableManager(through=TaggedItem, manager=_NautobotTaggableManager, ordering=["name"])
```

Comment:
I suspect I may have done this just to get generic create view prototype working.